source/settings/command-line-options.txt

```diff
@@ -75,6 +70,18 @@ configuration file.
        Specify a passphrase to decrypt the :ref:`imported favorite
        connections <compass-import-connections-cli-encrypted>` file. 
 
+   * - .. option:: --showExampleConfig
+     - Provides an example |compass-short| configuration file. 
+
+   * - .. option:: --trustedConnectionString
+     - Suppresses warnings about disallowed connection string properties and allows 
+       automatic connection. Compass inspects the connection string passed in the 
+       command line, and verifies whether it contains parameters that you may 
+       not want to have set and can result in a security risk. 
```

Comment:
```suggestion
       command line, and verifies whether the connection strings contains parameters that you may 
       not want to have set and can result in a security risk. 
```

left a small suggestion for clarity and also have a question: are "parameters that you may not want to have set" and "parameters that can result in a security risk" the same thing in this context? How does Compass know what parameters you may not want to have set? I wonder if we can make "parameters that you may not want to have to set" more clear

Reply:
No this makes more sense, thank you!